src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -10,6 +10,7 @@
 
   .accordion-button-text {
     font-size: 1.35em;
+    font-family: 'bitter', serif;
```

Comment:
Is there a reason that applying the `.vads-u-font-family--serif`class wouldn't work?  More info [here](https://design.va.gov/utilities/font-family).

Reply:
Applied these changes